ts/nni_manager/core/nnimanager.ts

```diff
@@ -661,6 +676,7 @@ class NNIManager implements Manager {
                     this.currSubmittedTrialNum++;
                     this.log.info('submitTrialJob: form:', form);
                     const trialJobDetail: TrialJobDetail = await this.trainingService.submitTrialJob(form);
+                    setTimeout(async ()=> this.stopTrialJobIfOverMaxDurationTimer(trialJobDetail.id), 1000 * this.maxTrialDuration);
```

Comment:
do you test if this works?

Reply:
Yes, it is works, with "maxTrialDuration" set and not set.